playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -436,3 +437,60 @@ def foreign_logs_enricher(event: ExecutionBaseEvent, params: ForeignLogParams):
 
 
 logs_enricher = action(logs_enricher)
+
+
+class MentionParams(ActionParams):
+    static_mentions: Optional[List[str]]
+    mentions_label: Optional[str]
+    message_template: Optional[str]
+
+
+@action
+def mention_enricher(event: KubernetesResourceEvent, params: MentionParams):
+    """
+    You can define who to mention using a static mentions configuration,
+    Or, you can define it using a label or annotation, that exists either on the Kubernetes resource, or the alert
+
+    Order:
+    1. Resource annotations (For alert, get from FindingSubject. For other resources, get from obj metadata)
+    2. Resource labels (For alert, get from FindingSubject. For other resources, get from obj metadata)
+    3. Alert annotations (only for alert)
+    4. Alert labels (only for alert)
+
+    Note this enricher only works with the Slack sink
+    """
+    event_data = {}
+    if params.mentions_label:
+        if isinstance(event, PrometheusKubernetesAlert):
+            # Alert labels and annotations. FindingSubject can represent
+            # e.g. a k8s pod, job, daemonset etc etc.
+            alert_subject: FindingSubject = event.get_alert_subject()
+            event_data.update(alert_subject.annotations)
+            event_data.update(alert_subject.labels)
+            event_data.update(event.alert.annotations)
+            event_data.update(event.alert.labels)
+        elif event.obj:
+            # TODO why do we ignore alert_subject here?
+            if event.obj.metadata.annotations:  # TODO why the if?
+                event_data.update(event.obj.metadata.annotations)
+            if event.obj.metadata.labels:  # TODO why the if?
+                event_data.update(event.obj.metadata.labels)
+
+        # get the mentions and use it
+        mentions_value = event_data.get(params.mentions_label)
+        if mentions_value:
+            users_list = mentions_value.split(",")
+        else:
+            users_list = []
+    elif params.static_mentions:
+        # TODO shouldn't we rather use the union of the mentions gathered above and the static ones?
+        users_list = params.static_mentions
+    else:
+        logging.warning("mention_enricher called with neither static_mentions nor mentions_label set")
+        return
+
+    # TODO should we additionally allow for setting messsage_template per finding subject etc as is
```

Comment:
I think that's not necessary
We have other enrichments for custom messages
This should be only for mentions I think